src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PivotTests/UnoSamples_Pivot_Tests.cs

```diff
@@ -26,5 +26,27 @@ public void Pivot_Non_PivotItem_Items()
 			_app.WaitForDependencyPropertyValue(selectedItemTitle, "Text", "item 1");
 			_app.WaitForDependencyPropertyValue(selectedItemContent, "Text", "My Item 1 Content");
 		}
+		[Test]
+		[AutoRetry()]
+		public void Pivot_Non_PivotItemChange_Validation()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.Pivot.Pivot_CustomContent_Automated");
+
+			var selectedItemTitle = _app.Marked("selectedItemTitle");
+			var selectedItemContent = _app.Marked("selectedItemContent");
+			var itemHeaderTextBlock = _app.Marked("ItemHeaderTextBlock");
+
+			// Select header 2 and assert
+			itemHeaderTextBlock.AtIndex(1).Tap();
```

Comment:
I'm not sure this works properly for Wasm. If it does not, add named `AutomationProperty.AutomationId="{Binding SomeID}"` and use marked selection.

See this example:
https://github.com/unoplatform/uno/blob/13af8f8ca9a19360d4acd6ae737dcc37a099a5ac/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/ListView_SelectedItem.xaml#L25

Reply:
Yes @jeromelaban AtIndex(int i) does not work on Wasm. I added like you suggested AutomationProperty.AutomationId in the XAML and adjusted the code for the test. After verifying Wasm and Android, the test is working fine now on both.